src/install.ts

```diff
@@ -129,6 +132,11 @@ class UI {
     return p;
   }
   set clangdPath(p: string) {
-    config.update('path', p, vscode.ConfigurationTarget.Global);
+    this._pathUpdated =
```

Comment:
I believe a more conventional way to write this is:

```js
this._pathUpdated = new Promise(resolve => {
  config.update('path', ...).then(resolve);
});
```

or alternatively:

```js
this._pathUpdated = new Promise(async resolve => {
  await config.update('path', ...);
  resolve();
});
```

This avoids the extra variable `_resolvePathUpdated`, and also ensures that if the setter is called multiple times, awaiting `_pathUpdated` will correctly wait until the most recent promise is resolved.

Reply:
I went with first option as `config.update` is not awaitable.